aquadoggo/src/db/provider.rs

```diff
@@ -33,13 +29,9 @@ impl SqlStorage {
 /// databases.
 #[async_trait]
 impl StorageProvider for SqlStorage {
-    type EntryArgsRequest = EntryArgsRequest;
-    type EntryArgsResponse = EntryArgsResponse;
-    type PublishEntryRequest = PublishEntryRequest;
-    type PublishEntryResponse = PublishEntryResponse;
     type StorageLog = StorageLog;
```

Comment:
Rename to Log

Reply:
Will do this in follow-up PR when I make some other storage provider api changes.